accurest-core/src/test/groovy/io/codearte/accurest/dsl/WiremockGroovyDslSpec.groovy

```diff
@@ -198,6 +198,314 @@ class WiremockGroovyDslSpec extends WiremockSpec {
 			stubMappingIsValidWiremockStub(wiremockStub)
 	}
 
+	def 'should use equalToJson when body match is defined as map'() {
```

Comment:
I'm wondering if we could build an XML with a map... In the future maybe first we'll add the possibility to provide `MarkupBuilder` support for XMLs.